tests/unit_tests/test_server_client.py

```diff
@@ -349,6 +349,109 @@ async def test_astream_log(async_client: RemoteRunnable) -> None:
     ]
 
 
+@pytest.mark.asyncio
+async def test_astream_log_allowlist(event_loop: AbstractEventLoop) -> None:
+    """Test async stream with an allowlist."""
+
+    async def add_one(x: int) -> int:
+        """Add one to simulate a valid function"""
+        return x + 1
+
+    app = FastAPI()
+    add_routes(
+        app,
+        RunnableLambda(add_one),
+        path="/empty_allowlist",
+        input_type=int,
+        stream_log_name_allowlist=["allowed"],
+    )
+    add_routes(
+        app,
+        RunnableLambda(add_one),
+        input_type=int,
+        path="/allowlist",
+        stream_log_name_allowlist=["allowed"],
+    )
+
+    async with get_async_client(app, path="/empty_allowlist") as runnable:
```

Comment:
The fact that this didn't end in a slash triggered the client error. Might be worth adding a test specifically for that.